pynuodb/encodedsession.py

```diff
@@ -393,9 +394,14 @@ def putScaledInt(self, value):
         @type value decimal.Decimal
         """
         #Convert the decimal's notation into decimal
-        value += 0
+        value += REMOVE_FORMAT
         scale = abs(value.as_tuple()[2])
         valueStr = toSignedByteString(int(value * decimal.Decimal(10**scale)))
+
+        #If our length is more than 9 bytes we will need to send the data using ScaledCount2
+        if len(valueStr) > 9:
```

Comment:
I don't understand this change. Let's talk more about it tomorrow. valueStr is a string of digits right? So we are sending edsScaledCount2 for a string longer than 9? Normally it takes I think 20 digits to need an edsScaledCount2. Maybe I am missing something.